src/LibraryManager.Vsix/UI/Models/LibraryIdViewModel.cs

```diff
@@ -23,7 +23,15 @@ public LibraryIdViewModel(ISearchService packageSearchService, string initialSea
         protected override Task<CompletionSet> FilterCompletions(CompletionSet input)
         {
             CompletionSet result = input;
-            int atIndex = SearchText.IndexOf('@');
+            int atIndex = SearchText.LastIndexOf('@');
+
+            //For scoped packages
```

Comment:
Finish the explanation:
`// For scoped packages, the version segment starts after an @ sign which follows a /`